src/Entities/Player.cpp

```diff
@@ -461,8 +461,13 @@ void cPlayer::SetTouchGround(bool a_bTouchGround)
 
 	static const auto HalfWidth = GetWidth() / 2;
 	static const auto EPS = 0.0001;
+
+	BLOCKTYPE BlockAtFoot = GetWorld()->GetBlock(POS_TOINT);
```

Comment:
Looking at `GetBlock` it will return 0, if there is no match found in the Chunk.